fastlane/Fastfile

```diff
@@ -59,6 +59,95 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases"
+lane :update_native_dependencies_to_latest do |options|
+  latest_ios_release = get_latest_github_release_name(repo_name: "RevenueCat/purchases-ios")
+  latest_android_release = get_latest_github_release_name(repo_name: "RevenueCat/purchases-android")
+
+  UI.message("ℹ️  Latest RevenueCat iOS version: #{latest_ios_release}")
+  UI.message("ℹ️  Latest RevenueCat Android version: #{latest_android_release}")
+
+  update_ios_native_version(version: latest_ios_release)
+  update_android_native_version(version: latest_android_release)
+
+  run_pod_install
+end
+
+desc "Update purchases-ios dependency version"
+lane :update_ios_native_version do |options|
```

Comment:
I think we can probably extract most of this logic to a function in this Fastfile so it can be shared with the android lane.

Reply:
+1